### PR TITLE
Create List Methods

### LINKED ISSUE #4: Implement List resource methods
## Overview 

We need to build out the API methods for interacting with Lists on Boards. This will enable full CRUD functionality for Lists.

## Tasks

- [x] `get_list()`
- [x] `create_list()`  
- [x] `update_list()`
- [x] `delete_list()` 
- [x] `get_cards_on_list()`

## Acceptance Criteria  

- Write tests for Board methods
- Document Board methods as doc strings

### PR DESCRIPTION
**Pull request by purplemoon-l**
All 5 API(s)
1. Create List
2. Get List
3. Get All Cards in the list
4. Archive List
5. Update List

Fixes #4

### CODE REVIEW

File: pytrello2/models/list.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+class Card:
```
Comment (vchrombie) [inline pytrello2/models/list.py:1]:
```suggestion
class List
```

File: pytrello2/models/list.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        self.idBoard = data.get("idBoard")
+        self.name = data.get("name")
+        self.url = data.get("url")
+
+    def __str__(self):
+        """
+        Returns a string representation of a List object.
+        """
+        return (
+            f"Card(id={self.id}, idBoard={self.idBoard}, name={self.name},"
```
Comment (vchrombie) [inline pytrello2/models/list.py:20]:
```suggestion
            f"List(id={self.id}, idBoard={self.idBoard}, name={self.name},"
```

**Review by vchrombie — CHANGES_REQUESTED** [review]

File: pytrello2/list.py [hunk 3]
```diff
@@ -0,0 +1,54 @@
+from .models.list import ListClass
```
Comment (vchrombie) [inline pytrello2/list.py:1]:
```suggestion
from .models.list import List
```

File: pytrello2/models/list.py [hunk 4]
```diff
@@ -0,0 +1,22 @@
+class ListClass:
```
Comment (vchrombie) [inline pytrello2/models/list.py:1]:
```suggestion
class List:
```

**Review by vchrombie — CHANGES_REQUESTED** [review]

File: pytrello2/list.py (hunk 3 above)
Comment (purplemoon-l) [reply pytrello2/list.py]: Resolved

**Review by vchrombie — APPROVED** [review]
LGTM! 🚀

**Pull request merged**